fcos-config.yaml

```diff
@@ -10,6 +10,34 @@ systemd:
    - name: podman.socket
      enabled: true
 storage:
+  # Ignition has a bug/feature? where if you make a series of dirs
+  # in one swoop, then the leading dirs are creates as root.
```

Comment:
'creates' -> 'created' 

'session' -> 'section' in the commit log, and 'subdirectiors' -> 'subdirectories'